src/Tribe/RSVP.php

```diff
@@ -1547,12 +1560,14 @@ public function get_tickets_ids( $event_id ) {
 	 * @param $event_id
 	 * @param $ticket_id
 	 *
+	 * @since TBD
+	 *
 	 * @return mixed
 	 */
 	public function do_metabox_capacity_options( $event_id, $ticket_id ) {
 
 		$stock = '';
-
+		// This returns the original stock?
```

Comment:
Rephrase so it doesn't read like a question (unless we're unsure, of course, in which case maybe we should discuss, clarify and remove the question)?